os/arch/Kconfig

```diff
@@ -764,6 +764,15 @@ config RAM_REGIONx_HEAP_INDEX
 		It means that its region can be handled to the index-th heap. 
 		Index can be from 0 to (MM_NHEAPS-1).
 
+config RAM_MALLOC_PRIOR_INDEX
+	int "prior region index of RAM used in malloc"
+	default 0
+	depends on MM_REGIONS != 1
```

Comment:
This can be from 0 to (MM_REGIONS-1). In this case, you can set the range in Kconfig.

Reply:
the dependency changed to MM_NHEAPS, this config is smiliar as RAM_REGIONx_HEAP_INDEX, the range is dynamic according to MM_NHEAPS, so it's not proper to add range for it.